src/nccl_ofi_sendrecv.c

```diff
@@ -1656,6 +1661,77 @@ static int send(nccl_net_ofi_send_comm_t *send_comm, void *data, int size, int t
 	return ret;
 }
 
+typedef struct {
+	nccl_ofi_freelist_reginfo_t reginfo;
+	uint64_t value;
+} sendrecv_write_inline_fl_item_t;
+
+typedef struct {
+	nccl_ofi_mr_keypool_t *key_pool;
+	void *mr_handle;
+	int dev_id;
+} sendrecv_write_inline_fl_handle_t;
+
+static int write_inline(nccl_net_ofi_comm_t *comm, void *data, int size, void *dest, void *mhandle)
```

Comment:
Does this function belong in the RDMA protocol rather than the SENDRECV protocol as it is here?